modules/xatlas_unwrap/register_types.cpp

```diff
@@ -141,11 +141,11 @@ bool xatlas_mesh_lightmap_unwrap_callback(float p_texel_size, const float *p_ver
 
 	xatlas::Atlas *atlas = xatlas::Create();
 	printf("Adding mesh..\n");
-	xatlas::AddMeshError::Enum err = xatlas::AddMesh(atlas, input_mesh, 1);
-	ERR_FAIL_COND_V_MSG(err != xatlas::AddMeshError::Enum::Success, false, xatlas::StringForEnum(err));
+	xatlas::AddMeshError err = xatlas::AddMesh(atlas, input_mesh, 1);
+	ERR_FAIL_COND_V_MSG(err != xatlas::AddMeshError::Success, false, xatlas::StringForEnum(err));
 
 	printf("Generate..\n");
-	xatlas::Generate(atlas, chart_options, xatlas::ParameterizeOptions(), pack_options);
+	xatlas::Generate(atlas, chart_options, xatlas::PackOptions());
```

Comment:
Someone who knows more than me, I don't fully understand this change. It made it build, but someone should check my work?

Reply:
Instead of using:

```

	pack_options.maxChartSize = 4096;
	pack_options.blockAlign = true;
	pack_options.padding = 1;
	pack_options.texelsPerUnit = 1.0 / p_texel_size;
```

You set it to the default. This is arguably a bug.

Reply:
You probably want `xatlas::Generate(atlas, chart_options, xatlas::pack_options);`